paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/WalletButtonsSettingsDefinition.kt

```diff
@@ -0,0 +1,26 @@
+package com.stripe.android.paymentsheet.example.playground.settings
+
+internal object WalletButtonsSettingsDefinition : BooleanSettingsDefinition(
+    key = "WalletButtons",
+    displayName = "Wallet Buttons",
+    defaultValue = false
+) {
+    override fun applicable(configurationData: PlaygroundConfigurationData): Boolean {
+        return configurationData.integrationType == PlaygroundConfigurationData.IntegrationType.Embedded
+    }
+
+    override fun createOptions(
```

Comment:
I think you can omit this if you change the name to be "Show Wallet Buttons" -- Then true/false make sense.